src/connections/ble_handler.rs

```diff
@@ -45,6 +45,16 @@ pub enum BleId {
 }
 
 impl BleId {
+    /// Constructs BLE ID from the short name used by Meshtastic.
+    ///
+    /// It ends with `_abcd`, where `abcd` are the last 4 hex-digits of the MAC address.
```

Comment:
I can't tell from this comment whether the 4-digit hex MAC is required here or not. Could you clarify this comment a bit?

Reply:
I don't have a meshtastic device with me right now, I'll do some tests and update the comment in a few days.

Reply:
Improved the comments